src/riakc_utils.erl

```diff
@@ -49,3 +54,8 @@ characters_to_unicode_binary(String) ->
         Binary ->
             Binary
     end.
+
+-spec to_binary(atom()|iolist()|binary()) -> binary().
+%% @doc Convert to binary, additionally supporting atom().
+to_binary(V) when is_atom(V) -> to_binary(atom_to_list(V));
+to_binary(V) -> iolist_to_binary(V).
```

Comment:
My first question was, should there be a clause for when V is (already) a binary? but it looks like iolist_to_binary can take a binary and is idempotent in that case.  Maybe guard the second clause?  I guess it is correct the way it is, so it's fine the way it is.